### ISSUE TITLE
Build error -- libtiff

### ISSUE THREAD

**Issue by absudabsu** [srounet/docker-mapnik#3]
ubuntu:latest does not have `libtiff4-dev`, must update to `libtiff5-dev`:

```
Step 8 : RUN apt-get -qq install --yes libicu-dev libtiff4-dev libfreetype6-dev libpng12-dev libxml2-dev libproj-dev libsqlite3-dev libgdal-dev libcairo-dev python-cairo-dev postgresql-contrib libharfbuzz-dev
 ---> Running in e294bf1163f1
E: Package 'libtiff4-dev' has no installation candidate
The command '/bin/sh -c apt-get -qq install --yes libicu-dev libtiff4-dev libfreetype6-dev libpng12-dev libxml2-dev libproj-dev libsqlite3-dev libgdal-dev libcairo-dev python-cairo-dev postgresql-contrib libharfbuzz-dev' returned a non-zero code: 100
```

I made the change in the dockerfile, and issued a pull request (#2). Seems to build fine for me now.

**Comment by srounet** [comment]
I try this and let you know, thank you for the PR, and keeping this project up-to-date.

I'll fix the Ubuntu version also in an other commit.

**Comment by srounet** [comment]
Thank you for the reporting, this is now fixed.
Mapnik is also bumped to **3.0.10** and Ubuntu is fixed to **16.04** to avoid any breaks in the future.

I mark this as resolved.
